byom/pima/evaluation.py

```diff
@@ -0,0 +1,80 @@
+from sklearn import metrics
```

Comment:
i added the byom evaluation example to this repository. I'll need to update the docs later but its a placeholder for them